package.json

```diff
@@ -74,7 +74,10 @@
     "test": "npm-run-all -p unit lint",
     "lint": "eslint \"*.?(c)js\" lib/ tests/lib/",
     "fixlint": "npm run lint -- --fix",
-    "rollup": "rollup -c rollup.config.js",
+    "build": "rollup -c rollup.config.js",
```

Comment:
We should update [ci.yml](https://github.com/eslint/espree/blob/main/.github/workflows/ci.yml), it uses `npm run rollup`.